helpers/filterSeats.js

```diff
@@ -60,19 +69,73 @@ module.exports = function(dust) {
       });
     }
 
+    // get the names of the best two sections from the theatre
+    function getBestSections() {
+      // add all section names to an array
+      _.forEach(bandColours, function(bands, section) {
+        bestSections.push(section);
+      });
+      // find the best section
+      bestSections = _.last(bestSections); // just Stalls
+    }
+
+    getBestSections();
+
+    // compare prices
+    function chosenForYou(ticketRate, ticketRateSection) {
+      // ticketRateSection
+      var ticketRateId = ticketRate.ids;
+      var ticketRateColour = ticketRate.colour;
+      var ticketRateColourRank = ticketRate.colourRank;
+      var ticketRatePrice = ticketRates[ticketRateId].grossPrice;
+      // check if current ticketRate exists in
+      if (bestSections === ticketRateSection && _.last(ticketRateColourRank)) {
+        // check if price is cheaper or exists
+        if (ticketRatePrice < currentPrice || !currentPrice) {
```

Comment:
I would flip this logic to be `!currentPrice || ticketRatePrice < currentPrice` as checking the existence of a thing is faster than comparing values otherwise if `currentPrice` is null you are still comparing THEN checking if it existed.